### PR TITLE
10.16.0 German translation user guide

### PR DESCRIPTION
**Pull request by MWindel**
New translations for the 10.16.0 German user guide.

* Replaced image links,
* variables,
* aliases

### CODE REVIEW

File: content/benutzerhandbuch/administration-de-bundle/data-retention.md [hunk 1]
```diff
@@ -23,12 +23,12 @@ Das Sternsymbol ("*") zeigt an, dass alle Daten, unabhängig vom jeweiligen Wert
 #### Datentypen
 
 Die Datenhaltungsregeln erstrecken sich auf folgende Datentypen:
-+ Alarme
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/benutzerhandbuch/administration-de-bundle/data-retention.md:26]: Done

File: content/benutzerhandbuch/administration-de-bundle/changing-settings.md [hunk 2]
```diff
@@ -589,9 +322,11 @@ Durch Bereitstellung Ihrer Zugangsdaten ermöglichen Sie die Nutzung von Plattfo
 
 1. Klicken Sie auf **SMS-Anbieter** im Menü **Einstellungen**.
 
-{{< c8y-admon-info >}}
+    ![Select SMS provider](/images/benutzerhandbuch/Administration/admin-settings-sms-provider.png)
+
+	{{< c8y-admon-info >}}
 Um die SMS-Anbieter-Konfiguration einsehen zu können, benötigen Sie die Berechtigung SMS LESEN. Um die SMS-Anbieter-Konfiguration ändern zu können, benötigen Sie die Berechtigung SMS ADMIN.
-{{< /c8y-admon-info >}}
+  {{< /c8y-admon-info >}}
```
Comment (MWindel) [inline content/benutzerhandbuch/administration-de-bundle/changing-settings.md:330]:
```suggestion
{{< /c8y-admon-info >}}
```

File: content/benutzerhandbuch/administration-de-bundle/changing-settings.md [hunk 3]
```diff
@@ -517,9 +248,11 @@ Benutzerdefinierte Attribute sind für alle authentifizierten Benutzer des Manda
 
 1. Wählen Sie die Registerkarte für das gewünschte Attribut und klicken Sie auf **Attribut hinzufügen**.
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/benutzerhandbuch/administration-de-bundle/changing-settings.md:250]: Done

File: content/benutzerhandbuch/cockpit-de-bundle/widgets-collection.md [hunk 4]
```diff
@@ -568,7 +572,7 @@ Das "Silo"-Widget zeigt Datenpunkte (Messwerte) mit aktuellen Werten in einer Si
 </tr>
 <tr>
 <td align="left">Datenpunkte</td>
-<td align="left">Zeigt eine Liste verfügbarer Datenpunkte. Sie müssen mindestens einen Datenpunkt aktivieren. Klicken Sie auf <strong>Datenpunkt hinzufügen</strong>, um einen Datenpunkt zur Liste hinzuzufügen. Informationen zum Hinzufügen von Datenpunkten finden Sie unter <a href="#add-data-points">Daten-Explorer &gt; Hinzufügen von Datenpunkten</a>.</td>
+<td align="left">Zeigt eine Liste verfügbarer Datenpunkte. Sie müssen mindestens einen Datenpunkt aktivieren. Klicken Sie auf <strong>Datenpunkt hinzufügen</strong>, um einen Datenpunkt zur Liste hinzuzufügen. Informationen zum Hinzufügen von Datenpunkten finden Sie unter <a href="#add-data-points">Daten-Explorer &gt; Zum Hinzufügen eines Datenpunkts</a>.</td>
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/benutzerhandbuch/cockpit-de-bundle/widgets-collection.md:575]:
```suggestion
<td align="left">Zeigt eine Liste verfügbarer Datenpunkte. Sie müssen mindestens einen Datenpunkt aktivieren. Klicken Sie auf <strong>Datenpunkt hinzufügen</strong>, um einen Datenpunkt zur Liste hinzuzufügen. Informationen zum Hinzufügen von Datenpunkten finden Sie unter <a href="#add-data-points">Daten-Explorer &gt; So fügen Sie einen Datenpunkt hinzu</a>.</td>
```

File: content/benutzerhandbuch/device-management-de-bundle/device-details.md [hunk 5]
```diff
@@ -279,7 +279,7 @@ Die Loginformationen des Geräts werden abgefragt.
 
 ![Logs tab](/images/benutzerhandbuch/DeviceManagement/devmgmt-devices-logs.png)
 
-Das Abfragen der Logdaten von einem Gerät kann einige Zeit in Anspruch nehmen.
+>Das Abfragen der Logdaten von einem Gerät kann einige Zeit in Anspruch nehmen.
```
Comment (MWindel) [inline content/benutzerhandbuch/device-management-de-bundle/device-details.md:282]:
```suggestion
Das Abfragen der Logdaten von einem Gerät kann einige Zeit in Anspruch nehmen.
```

File: content/benutzerhandbuch/device-management-de-bundle/connecting-devices.md [hunk 6]
```diff
@@ -53,10 +53,17 @@ Je nach Gerätetyp sind möglicherweise nicht alle beschriebenen Schritte releva
...
+
+Der Dialog **Geräte registrieren** wird angezeigt.
+
+  <img src="/images/benutzerhandbuch/DeviceManagement/devmgmt-registration-general.png" alt="General device registration" style="max-width: 100%">
+
 3. Geben Sie im Feld **Geräte-ID** die eindeutige ID des Geräts ein. Diese finden Sie in der Gerätedokumentation. Bei mobilen Geräten handelt es sich dabei meistens um die IMEI (International Mobile Equipment Identity), die häufig auf der Rückseite des Geräts zu finden ist.
 4. Wählen Sie optional eine Gruppe aus, der Sie das Gerät nach der Registrierung zuweisen möchten, siehe auch [Gruppieren von Geräten](#grouping-devices).
-5. Klicken Sie auf **Neues Gerät hinzufügen**, um ein weiteres Gerät zu registrieren. Geben Sie auch hier die Geräte-ID ein und weisen Sie optional das Gerät einer Gruppe zu. Auf diese Weise können Sie mehrere Geräte in einem Schritt registrieren.
+5. Klicken Sie auf **Weiteres Gerät hinzufügen**, um ein weiteres Gerät zu registrieren. Geben Sie auch hier die Geräte-ID ein und weisen Sie optional das Gerät einer Gruppe zu. Auf diese Weise können Sie mehrere Geräte in einem Schritt registrieren.
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/benutzerhandbuch/device-management-de-bundle/connecting-devices.md:66]:
```suggestion
5. Klicken Sie auf **Gerät hinzufügen**, um ein weiteres Gerät zu registrieren. Geben Sie auch hier die Geräte-ID ein und weisen Sie optional das Gerät einer Gruppe zu. Auf diese Weise können Sie mehrere Geräte in einem Schritt registrieren.
```

File: content/benutzerhandbuch/administration-de-bundle/audit-logs.md [hunk 7]
```diff
@@ -27,12 +27,12 @@ Um die Audit-Logs-Liste anzuzeigen, klicken Sie auf **Audit-Logs** im Menü **Ko
 
 <tr>
 <td align="left">Ereignis</td>
-<td align="left">Typ der Operationen, z. B. "Alarm erstellt", "Smart Rule gelöscht". Darunter wird der Benutzer angezeigt, der die Operation verarbeitet hat.</td>
+<td align="left">Typ der Operationen, z. B. "Alarm erstellt", "Smart Rule gelöscht". Darunter wird der Benutzer angezeigt, der die Operation verarbeitet hat.</td>
```
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/benutzerhandbuch/administration-de-bundle/audit-logs.md:30]: Aber in den Admonitions, also in den Admonition Texten, hatte das ja genau zu dem Problem geführt, oder nicht?

File: content/benutzerhandbuch/device-management-de-bundle/connectivity.md [hunk 8]
```diff
@@ -78,19 +78,21 @@ Führen Sie den folgenden Schritt aus, um die Konnektivität in {{< product-c8y-
...
 2. Wechseln Sie zur Registerkarte **Mobilfunkanbieter-Einstellungen**.
 3. Wählen Sie einen Anbieter aus der Auswahlliste aus.
-1. Geben Sie die Zugangsdaten (URL, Schlüssel (im Falle von Jasper), Benutzername und Passwort) für das entsprechende SIM-Anbieter-Konto ein. Wenn Ihnen keine Zugangsdaten vorliegen, fragen Sie Ihren Administrator.
-2. Klicken Sie auf **Speichern**, um Ihre Einstellungen zu speichern.
+1. Geben Sie die Zugangsdaten für das entsprechende SIM-Anbieter-Konto ein. Wenn Ihnen keine Zugangsdaten vorliegen, fragen Sie Ihren Administrator.
+2. Legen Sie eine **Cache duration** (Cache-Dauer) in Sekunden fest um zu bestimmen, wie lange die vom Anbieter stammenden Informationen zwischengespeichert werden sollen. Dadurch werden Timeout-Probleme vermieden.
```
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/benutzerhandbuch/device-management-de-bundle/connectivity.md:82]: I remember somehow that I asked Pawel about that during your vacation, probably the translators asked for it. Ok, let´s leave as is then.

File: content/benutzerhandbuch/cockpit-de-bundle/managing-assets.md [hunk 9]
```diff
@@ -4,6 +4,14 @@ title: Verwalten von Assets
 weight: 20
 ---
 
+Klicken Sie im Navigator auf **Gruppen**, um eine Liste aller Gruppen zu sehen. Um eine Gruppe hinzuzufügen, klicken Sie oben rechts auf **Gruppe hinzufügen**.
```
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/benutzerhandbuch/cockpit-de-bundle/managing-assets.md:7]: You are right, sorry, my fault.

File: content/benutzerhandbuch/administration-de-bundle/audit-logs.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (MWindel) [reply content/benutzerhandbuch/administration-de-bundle/audit-logs.md:30]: Die hat Richard im commit [6c9deca](https://github.com/SoftwareAG/c8y-docs/pull/874/commits/6c9deca946566a968458cad5364cf681ba3eb3fb) und [de2f4f7](https://github.com/SoftwareAG/c8y-docs/pull/874/commits/de2f4f76e45ae39fc48b21273c5a7885c984e5dc) gefixt.

File: content/benutzerhandbuch/administration-de-bundle/managing-permissions.md [hunk 10]
```diff
@@ -370,12 +370,37 @@ Wenn ein Benutzer Stammdatenzugriff für eine Gerätegruppe hat, hat er auch Zug
...
+* Seiten mit aggregierten Ereignissen von mehreren Geräten, wenn die Anzahl der Ereignisse gering ist, z. B. [Device Management > Überwachen und Steuern von Geräten > So zeigen Sie Ereignisse an](/benutzerhandbuch/device-management-de/#to-view-events)
+* Seiten mit aggregierten Operationen von mehreren Geräten, wenn die Anzahl der Operationen gering ist, z. B. [Device Management > Übersichten > Gerätesteuerung](/benutzerhandbuch/device-management-de/#to-view-single-operations)
+
+Als Administrator können Sie die Leistungsfunktion auf folgende Weise deaktivieren:
+- Auf Plattformebene über die Konfigurationsdatei (nur für Plattformadministratoren verfügbar, siehe *{{< product-c8y-iot >}} - Operations Guide*).
+- Auf Mandantenebene über eine Mandantenoption. Die Mandantenoption hat zwei mögliche Werte: LEGACY/OPTIMIZED, wobei OPTIMIZED die globale Standardeinstellung ist.
+
+Die Option sieht in der REST API wie folgt aus (siehe auch [{{< openapi >}}](https://cumulocity.com/api/10.16.0/#operation/postOptionCollectionResource)):
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/benutzerhandbuch/administration-de-bundle/managing-permissions.md:396]:
```suggestion
Die Option sieht in der REST API wie folgt aus (siehe auch [{{< openapi >}}](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#operation/postOptionCollectionResource)):
```

**Pull request merged**